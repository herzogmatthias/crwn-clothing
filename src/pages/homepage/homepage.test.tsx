import React from "react";
import { shallow } from "enzyme";
import HomePage from "./homepage.component";

describe("HOMEPAGE TESTS", () => {
  let wrapper = shallow(<HomePage></HomePage>);
  test("render Homepage", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
