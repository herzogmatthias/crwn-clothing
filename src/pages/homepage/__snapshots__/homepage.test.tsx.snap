// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HOMEPAGE TESTS render Homepage 1`] = `
<styled.div>
  <Connect(Directory) />
</styled.div>
`;
